images/nginx/rootfs/build.sh

```diff
@@ -75,9 +75,6 @@ get_src()
   rm -rf "$f"
 }
 
-apk update
```

Comment:
Is this not necessary anymore since we're already on the latest version

Reply:
This was move outside the build script https://github.com/kubernetes/ingress-nginx/pull/6760/files#diff-78fbab3655980c4b1ba1cb27363d5eebcde239453bf026b9c4ec48977f057a57R20